src/lib/getKeyDownHandlers.ts

```diff
@@ -34,9 +42,20 @@ const getKeyDownHandlers = (
     keyHandlers[componentPart] = {
       onKeyDown: (event: KeyboardEvent) => {
         handledActions.forEach(actionName => {
+          let keyCombinations = componentPartKeyAction[actionName].keyCombinations
+
+          if (isRtl) {
+            keyCombinations = keyCombinations.map(keyCombination => {
+              if (rtlKeyMap[keyCombination.keyCode]) {
+                keyCombination.keyCode = rtlKeyMap[keyCombination.keyCode]
+              }
+              return keyCombination
```

Comment:
create variable for `rtlKeyMap[keyCombination.keyCode]` and reuse it when assigning it to the keyCode